### ISSUE TITLE
Clarify the semantics of tearing off an inherited abstract method

### ISSUE THREAD

**Issue by bwilkerson** [dart-lang/sdk#33951]
I have the following code:
```
abstract class A {
  void test();
}
class B extends A {
  void test() {
    super.test; // code in question
  }
}
```
Dart analyzer does not produce any static errors, but the CFE is currently issuing an error indicating that the class `A` does not define a getter named `test`. I couldn't find any place in the spec that would indicate that this should be an error. In particular, section 16.19.2 "Super Getter Access and Method Closurization" appears to indicate that this should be valid.

Should this code produce an error?

If so, then I'll open an analyzer issue. If not, then I'll open a CFE issue.

**Comment by stefantsov** [comment]
Note that CFE only produces error if class `B` is not `abstract` and `enableSuperMixins` is `false`; it compiles the program successfully otherwise.

**Comment by leafpetersen** [comment]
Yes, this should be an error.  See also https://github.com/dart-lang/sdk/issues/33662 .

**Comment by leafpetersen** [comment]
Filed a spec bug to make sure the text reflects this: https://github.com/dart-lang/sdk/issues/33963 .

**Comment by eernstg** [comment]
Fixed in #33963.
